be/src/exprs/agg/aggregate.h

```diff
@@ -107,6 +107,7 @@ class AggregateFunction {
     virtual bool is_pod_state() const { return false; }
     virtual void create(FunctionContext* ctx, AggDataPtr __restrict ptr) const = 0;
     virtual void destroy(FunctionContext* ctx, AggDataPtr __restrict ptr) const = 0;
+    virtual void init(FunctionContext* ctx, AggDataPtr __restrict ptr) const = 0;
```

Comment:
it seems it unnecessary. it won't be called in NullableAggregateFunction

Reply:
we prefer call init before call batch_update

Reply:
why unnecessary? it is actually called in AggregationFunction in this PR.
`batch_update` will invoke it for each batch, which is unexpectecd.